docker-compose.yml

```diff
@@ -3,22 +3,22 @@
 # Name: docker-compose.yml
 #
 # Function:
-# 	Configure the docker-iot-dashboard collection of docker containers.
```

Comment:
I don't know why the indentation changed. If you substituted spaces for tabs, you had tab stop set incorrectly. When you change tabs to spaces, it's important to maintain visual consistency, unless there's a good reason not to.

Reply:
I shouldn't have changed the format. It's corrected now.